server/tserver/src/main/java/org/apache/accumulo/tserver/TabletIteratorEnvironment.java

```diff
@@ -50,14 +52,16 @@
   private SamplerConfiguration samplerConfig;
   private boolean enableSampleForDeepCopy;
 
-  public TabletIteratorEnvironment(IteratorScope scope, AccumuloConfiguration config) {
+  public TabletIteratorEnvironment(IteratorScope scope, AccumuloConfiguration config,
```

Comment:
I don't think this constructor needs to be modified because it not intended to be called when scope is majc.

Reply:
@matthpeterson  do you have thoughts on this?